javascript/HtmlEditorField.js

```diff
@@ -1104,6 +1121,26 @@ ss.editorWrappers['default'] = ss.editorWrappers.tinyMCE;
 			}
 		});
 
+		//When 'Insert from URL' button is clicked
+		$('form.htmleditorfield-mediaform div.ss-upload .upload-url').entwine({
+			onclick: function () {
+				var form = this.closest('form');
+
+				form.addClass('insertingURL');
+				form.redraw();
+			}
+		});
+
+		//When back button is clicked while inserting URL
+		$('form.htmleditorfield-mediaform .htmleditorfield-mediaform-heading .font-icon-left-open').entwine({
```

Comment:
This is mixing presentation with behaviour - it should be applied to a class like `.back-button` rather than `.font-icon-left-open`